photon/photon_test.go

```diff
@@ -104,13 +104,32 @@ func TestConfig_Update(t *testing.T) {
 			appFs: afero.NewMemMapFs(),
 			bzip2FileNames: map[string]string{
 				"/photon_cve_metadata/photon_versions.json":    "testdata/photon_versions.json",
-				"/photon_cve_metadata/cve_data_photon1.0.json": "testdata/cve_data_photon1.0.json",
-				"/photon_cve_metadata/cve_data_photon2.0.json": "testdata/cve_data_photon2.0.json",
-				"/photon_cve_metadata/cve_data_photon3.0.json": "testdata/cve_data_photon3.0_invalid_cveid.json",
+				"/photon_cve_metadata/cve_data_photon1.0.json": "testdata/cve_data_photon3.0_invalid_cveid.json",
```

Comment:
I know it does not match, but this is to ensure this case fails early. other options:
1. create `testdata/cve_data_photon1.0_invalid_cveid.json` and set
```
...
"/photon_cve_metadata/cve_data_photon1.0.json": "testdata/cve_data_photon1.0_invalid_cveid.json",
...
```
2. or skip golden files for both 1.0 and 2.0 with empty_cveid
```
...
"/photon_cve_metadata/cve_data_photon1.0.json": "testdata/cve_data_photon_empty_cveid.json",
"/photon_cve_metadata/cve_data_photon2.0.json": "testdata/cve_data_photon_empty_cveid.json",
"/photon_cve_metadata/cve_data_photon3.0.json": "testdata/cve_data_photon3.0_invalid_cveid.json",
...
```